tools/wpt/testfiles.py

```diff
@@ -334,14 +336,13 @@ def get_parser():
     parser = argparse.ArgumentParser()
     parser.add_argument("revish", default=None, help="Commits to consider. Defaults to the "
                         "commits on the current branch", nargs="?")
-    # TODO: Consolidate with `./wpt run --affected`:
-    # https://github.com/web-platform-tests/wpt/issues/14560
-    parser.add_argument("--ignore-rules", nargs="*", type=set,  # type: ignore
-                        default={"resources/testharness*"},
-                        help="Rules for paths to exclude from lists of changes. Rules are paths "
-                        "relative to the test root, with * before a separator or the end matching "
-                        "anything other than a path separator and ** in that position matching "
-                        "anything")
+    parser.add_argument("--ignore-rule", action="append",
+                        help="Override the rules for paths to exclude from lists of changes. "
+                        "Rules are paths relative to the test root, with * before a separator "
```

Comment:
Does setting `default=DEFAULT_IGNORE_RULES` here not work as expected?

Reply:
No. Well, it depends on which semantics we want.

If we have a default combined the "append" action, new rules will add to default rules instead of overriding the default rules. I find overriding to be the more useful behaviour as you can always specify the default rules again.

Reply:
Makes sense.